rules/detekt/src/test/kotlin/com/twitter/compose/rules/detekt/ComposeModifierWithoutDefaultCheckTest.kt

```diff
@@ -41,6 +41,10 @@ class ComposeModifierWithoutDefaultCheckTest {
                     @Composable
                     fun Something(modifier: Modifier)
                 }
+                class BlehImpl : Bleh {
```

Comment:
You should add this too to the same test in rules/detekt/src/test/kotlin/com/twitter/compose/rules/ktlint/ComposeModifierWithoutDefaultCheckTest.kt